frontend/src/metabase/dashboard/components/ParametersPopover.unit.spec.tsx

```diff
@@ -0,0 +1,50 @@
+import { render, screen, within } from "__support__/ui";
+
+import { ParametersPopover } from "./ParametersPopover";
+
+const options = [
+  {
+    title: "Time",
+    subtitle: "Date range, relative date, time of day, etc.",
+    icon: "calendar",
+  },
+  {
+    title: "Location",
+    subtitle: "City, State, Country, ZIP code.",
+    icon: "location",
+  },
+  {
+    title: "ID",
+    subtitle: "User ID, Product ID, Event ID, etc.",
+    icon: "label",
+  },
+  {
+    title: "Number",
+    subtitle: "Subtotal, Age, Price, Quantity, etc.",
+    icon: "number",
+  },
+  {
+    title: "Text or Category",
+    subtitle: "Name, Rating, Description, etc.",
+    icon: "label",
+  },
+];
+
+describe("ParameterPopover", () => {
+  it("should render options with icons", () => {
+    render(
+      <ParametersPopover onClose={jest.fn()} onAddParameter={jest.fn()} />,
+    );
+
+    options.forEach(({ title, subtitle, icon }) => {
```

Comment:
it.each?

Reply:
[`1b57e07` (#44025)](https://github.com/metabase/metabase/pull/44025/commits/1b57e07ad64dca53f122ebc05f63b85391787f39)